cmake/Translations.cmake

```diff
@@ -184,7 +184,7 @@ macro (add_translations_catalog NLS_PACKAGE)
     
 
     # We need to create the directory if one extra content exists.
-    IF((NOT "${ARGS_DESKTOP_FILES}" STREQUAL "") OR (NOT "${ARGS_APPDATA_SOURCE}" STREQUAL "") OR (NOT "${ARGS_SCHEMA_SOURCE}" STREQUAL ""))
+    IF((NOT "${ARGS_DESKTOP_FILES}" STREQUAL "") OR (NOT "${ARGS_APPDATA_FILES}" STREQUAL "") OR (NOT "${ARGS_SCHEMA_FILES}" STREQUAL ""))
```

Comment:
Can we get an explanation on why this has changed?

Reply:
https://github.com/elementary/cmake-modules/pull/5